### DISCUSSION TITLE
How to config { javascriptEnabled: true } in quasar?

### DISCUSSION THREAD

**Discussion by 332lfgb6** [quasarframework/quasar#9333] [CLI - Electron mode] [answered]
Hi.I need to config  { javascriptEnabled: true } in quasar, normally, in a Vue CLI project, I configure it this way:
![image](https://user-images.githubusercontent.com/51245335/118776686-d3b1ba00-b8ba-11eb-9739-483ab551b377.png)
 but in quasar.config.js, CSS is an array rather than an object.
```js
// https://quasar.dev/quasar-cli/quasar-conf-js#Property%3A-css
css: [
  'app.scss',
  'common.less'
],
```
What should I do ?

**Comment by ontwikkelfabriek** [comment]
They are the  not the same `thing`. 

The `css` array in quasar.config.js  contains the global css files. ( excluding the default Quasar style files )
https://quasar.dev/quasar-cli/quasar-conf-js#options-to-configure

>  Hi.I need to config { javascriptEnabled: true } in quasar

Why do you think you need to do this in Quasar?

**Reply by 332lfgb6** [reply]
Sometimes need, I have solved, see his reply.

**Comment by TobyMosque** [comment, marked as answer]
https://quasar.dev/quasar-cli/quasar-conf-js#property-build
there is a build `options` called `lessLoaderOptions`

```js
build: {
  lessLoaderOptions: {
    javascriptEnabled: true
  }
}
```

however, u can alter any webpack config thought the `chainWebpack`
```js
chainWebpack (chain) {
  chain.module.rule('less').use('less').loader('less-loader').tag(opts => {
    return Object.sign(opts, { javascriptEnabled: true })
  })
}
```

**Reply by 332lfgb6** [reply]
**Thanks! I solved it with the first solution**
The first solution needs to be modified to look like this:
```js
lessLoaderOptions: {
  lessOptions: {
    javascriptEnabled: true
  }
}
```
The second solution reported an error:
![image](https://user-images.githubusercontent.com/51245335/118911953-9c461a80-b959-11eb-8e0e-6da9a8039ae7.png)
